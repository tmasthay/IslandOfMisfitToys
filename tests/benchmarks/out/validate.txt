              Relative Pointwise Difference              
┏━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━┓
┃          vp ┃        loss ┃         out ┃    out_filt ┃
┡━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━┩
│  0.0000e+00 │  0.0000e+00 │  0.0000e+00 │  0.0000e+00 │
│  6.6919e-07 │  1.0304e-04 │  2.2961e-05 │  1.9671e-05 │
│  4.7049e-06 │  4.3389e-04 │  1.7450e-04 │  9.9306e-05 │
│  1.1075e-04 │  2.0917e-02 │  3.0330e-03 │  1.6353e-03 │
│  2.6414e-04 │  2.6893e-02 │  4.8664e-03 │  4.3395e-03 │
│  2.2312e-04 │  1.8011e-02 │  2.8663e-03 │  1.7266e-03 │
│  4.7660e-04 │  3.2493e-02 │  4.5717e-03 │  2.7327e-03 │
│  3.2761e-04 │  2.3971e-02 │  2.5198e-03 │  1.6900e-03 │
│  2.3691e-04 │  8.3881e-03 │  1.7947e-03 │  1.4854e-03 │
│  2.1056e-04 │  6.6054e-03 │  1.1500e-03 │  9.1993e-04 │
└─────────────┴─────────────┴─────────────┴─────────────┘
