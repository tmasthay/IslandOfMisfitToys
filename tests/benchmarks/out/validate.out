              Relative Pointwise Difference              
┏━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━┓
┃          vp ┃        loss ┃         out ┃    out_filt ┃
┡━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━┩
│  0.0000e+00 │  0.0000e+00 │  0.0000e+00 │  0.0000e+00 │
│  1.0369e-05 │  1.3214e-03 │  3.1503e-04 │  1.7118e-04 │
│  1.5457e-05 │  1.5607e-03 │  5.4257e-04 │  3.0386e-04 │
│  4.0918e-04 │  4.3466e-02 │  7.8772e-03 │  3.3970e-03 │
│  1.6197e-04 │  1.3195e-02 │  4.2201e-03 │  4.1327e-03 │
│  1.0576e-04 │  8.8342e-03 │  1.9958e-03 │  1.3910e-03 │
│  1.4105e-04 │  8.3065e-03 │  3.2624e-03 │  2.6313e-03 │
│  1.5516e-04 │  2.3100e-02 │  2.5217e-03 │  1.6808e-03 │
│  1.6247e-04 │  4.6926e-02 │  3.6941e-03 │  4.0952e-03 │
│  1.0738e-04 │  5.7045e-04 │  2.1488e-03 │  1.8206e-03 │
└─────────────┴─────────────┴─────────────┴─────────────┘
